defaults/main.yml

```diff
@@ -19,4 +19,9 @@ nbde_client_provider: clevis
 #        - http://server2.example.com
 nbde_client_bindings: []
 
+# Settings used configuring dracut
+nbde_client_dracut_settings:
+  - kernel_cmdline="rd.neednet=1"
+  - ip="dhcp"
```

Comment:
I don't think it's a good idea to pass `ip="dhcp"` explicitly, as it may cause problems for people setting up network manually.

Reply:
Should this be added as an example but commented out or just removed I don't believe at actually required.
In which case should the rd.neednet be set to on by default?

Reply:
Set `rd.neednet=1 by default, since the role requires networking in the initramfs, but remove `ip=dhcp` as it may cause issues for configurations that already pass `ip=` to e.g. specify a static IP.